index.html

```diff
@@ -0,0 +1,135 @@
+    <link href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.13.0/css/all.min.css" rel="stylesheet">
+                
+</head>
+<body>
+    <header>
+        <span><img id="open-nav" src="./images/openNav.png" alt="open bar"></span>
+        <div class="upperNav">
+            <a href=""><img src="./images/facebook.png" alt="facebook logo" class="facebook"></a>
+            <a href=""><img class="twitter" src="./images/twitter.png" alt="twitter logo"></a>
+            <a href="">English</a>
+            <a href="">My Page</a>
+            <a href="">Logout</a>
+        </div>
+        <nav class="navbar">
+            <a href="#homepage" class="logo"><img src="./images/logo.png" alt="logo"></a>
+            <ul class="nav-links">
+                <li class="links-items active"><a href="about.html">About</a></li>
+                <li class="links-items"><a href="#program">Program</a></li>
+                <li class="links-items"><a href="#join">Join</a></li>
+                <li class="links-items"><a href="#sponsor">Sponsor</a></li>
+                <li class="links-items"><a href="#neaws">News</a></li>
+                <li class="links-items apart"><a href="#cc-campaign">CC Campaign</a></li>
+            </ul>
+        </nav>
+        <!-- Mobile menu hamburger -->
+        <div class="mobile-menu">
+            <div id="myNav" class="overlay">
+                <img id="close-nav" src="./images/closeNav.png" alt="close bar">
+                <div class="overlayContent">
+                    <a href="about.html" class="mobile-link">About</a>
+                    <a href="#program" class="mobile-link">Program</a>
+                    <a href="#join" class="mobile-link">Join</a>
+                    <a href="#sponsor" class="mobile-link">Sponsor</a>
+                    <a href="#news" class="mobile-link">News</a>
+                    <a href="#cc-campaign" class="mobile-link">CC Campaign</a>
+                </div>
+            </div>
+        </div>
+    </header>
+    <main>
+        <div class="header-container">
+            <div class="intro" id="homepage">
+                <h3>"Hello! Music World"</h3>
+                <h1>Musical Conference <br>Global Summit <br>2022</h1></span>
+            </div>
+            <div class="discription">
+                <p>Music conferences are invaluable when looking to learn and to grow in the industry. This a big opportunity to learn and to explore more on music. </p>
+            </div>
+            <div class="date">
+                <span id="date">2022.11.29(TUE) ~ 30(WED)</span>
+                <p>@ Art and Culture Center / Hollywood Florida
+                </p>
+            </div>
+   
+        </div>
+        <article class="main-program" id="program">
+            <a id="main-header" href="#">Main Program</a>
+            <hr>
+            <div class="programs" id="program">
+                <div class="programs-container">
+                <div class="program">
```

Comment:
- [ ] Kindly work on the card on the home page see picture below
your design is at the left and the template by the right.

<img width="1432" alt="Screenshot 2022-12-03 at 00 13 58" src="https://user-images.githubusercontent.com/81444338/205432330-68e91b54-ae98-4cac-94f4-5c8d80b92d43.png">

Reply:
Thanks. I have worked on it.